mod-linked-data/src/test/java/org/folio/CreateResourceSimulation.scala

```diff
@@ -1,41 +0,0 @@
-package org.folio
```

Comment:
Let us not remove this file. This file is for running gatling preformance test.

Here is the command for running the test -> `mvn clean gatling:test -Dkarate.env=ecs -pl common,testrail-integration,mod-linked-data`

Tests will create a directory named `target/gatling`. There you can see the test results.

I just executed the test and it failed. We can fix it in a separate ticket. However, let us not delete the file.

Reply:
done